File: Python-Programing-Basics/Conditional Statements Advanced/Conditional Statements Advanced - Lab/11_fruit_shop/fruit_shop.py
fruit = input()
day_of_week = input()
quantity = float(input())
fruits = ['banana', 'apple', 'orange', 'grapefruit', 'kiwi', 'pineapple', 'grapes']
days_of_week = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"]
working_days_price = [2.5, 1.2, 0.85, 1.45, 2.7, 5.5, 3.85]
rest_days_price = [2.7, 1.25, 0.9, 1.6, 3.0, 5.6, 4.2]

if day_of_week in days_of_week[:5] and fruit in fruits:
    index = fruits.index(fruit)
    price = working_days_price[index] * quantity
    print(f"{price:.2f}")
elif day_of_week in days_of_week[5:] and fruit in fruits:
    index = fruits.index(fruit)
    price = rest_days_price[index] * quantity
    print(f"{price:.2f}")
else:
    print("error")

File: JavaScript/Programming Basics - JavaScript/Conditional Statements Advanced/fruitShop.js
function getPrice (input) {
    let product = input[0];
    let day = input[1];
    let quantity = input[2];
    let price = 0;
    if (day === 'Sunday' || day === "Saturday") {
        switch(product){
            case 'banana': price = 2.70; break;
            case 'apple': price = 1.25; break;
            case 'orange': price = 0.90; break;
            case 'grapefruit': price = 1.60; break;
            case 'kiwi': price = 3.00; break;
            case 'pineapple': price = 5.60; break;
            case 'grapes': price = 4.20; break;
            default: price = 'error'; break;
        }
    }else if (day === 'Monday' || day === 'Tuesday' || day === 'Wednesday' || day === 'Thursday' || day === 'Friday'){
        switch(product){
            case 'banana': price = 2.50; break;
            case 'apple': price = 1.20; break;
            case 'orange': price = 0.85; break;
            case 'grapefruit': price = 1.45; break;
            case 'kiwi': price = 2.70; break;
            case 'pineapple': price = 5.50; break;
            case 'grapes': price = 3.85; break;   
            default: price = 'error'; break;
        } 
    } else {
        price = 'error';
    }
    if (price === 'error') {
        console.log(price)
    } else {
        console.log((price * quantity).toFixed(2));
    }
}